pkg/cache/v3/simple.go

```diff
@@ -527,8 +527,8 @@ func (cache *snapshotCache) respondDelta(ctx context.Context, snapshot ResourceS
 	// otherwise, envoy won't complete initialization
 	if len(resp.Resources) > 0 || len(resp.RemovedResources) > 0 || (state.IsWildcard() && state.IsFirst()) {
 		if cache.log != nil {
-			cache.log.Debugf("node: %s, sending delta response with resources: %v removed resources %v wildcard: %t",
-				request.GetNode().GetId(), resp.Resources, resp.RemovedResources, state.IsWildcard())
+			cache.log.Debugf("node: %s, sending delta response with wildcard: %t",
```

Comment:
I think it would still be useful to get a sense of what resources are being changed in this debug log, would be nice to have the typeurl and names of resources or at the very least how many were requested/removed

Reply:
@sunjayBhatia is there an existing helper method in the repo that converts `resources` to its name and typedUrl, or would I need to create one ?

Reply:
+1 for at least keeping the names. We can definitely remove the entire payload but let's keep the names for debugging purposes.

Reply:
the typeurl you should get with `request.TypeUrl`, `resp.RemovedResources` should give you the names of the computed resources to remove, and for the resources that are to be included in the response, as far as i know there isn't a direct helper, but there is a helper to get a resource name without fully marshalling it, see: https://github.com/envoyproxy/go-control-plane/blob/b9e8d765366d34116028760e6192d40d69628636/pkg/cache/v3/cache.go

we may want to instead of storing as a slice store these as a map? so its easier to get the names out?

a message like the below would be great:

```
node: %s, sending delta response for typeURL %s with resources: %v removed resources %v wildcard: %t
```